app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_COMP_CHAIN_DMA=y
 CONFIG_COMP_CROSSOVER=y
```

Comment:
I had to read the commit a few times, then code, and then again the commit, to understand what this is about. I think what confuses is the generic "tester framework" language. I'd actually not use the term "framework" here. IOW, in the end you are adding an audio module that happens to be implemented in a way that allows for easy extension. It would be maybe easier to understand if you just write: "Tester is an audio module that can be configured via module IPC to perform various test functions." 

I also wonder about the "tester" name. It is very generic. To add something specific (like load generation), a "loadgen" module would be more descriptive, but I guess you want to keep this generic on purpose.

Reply:
@kv2019i agree with you. It was very hard for me to understand what was all this about. And still will need to get back few times to read it.